x-pack/plugins/elastic_assistant/server/lib/langchain/execute_custom_llm_chain/index.ts

```diff
@@ -55,7 +55,8 @@ export const callAgentExecutor = async ({
     );
   }
 
-  const chain = RetrievalQAChain.fromLLM(llm, esStore.asRetriever());
+  // Create a chain that uses the ELSER backed ElasticsearchStore, override k=10 for esql query generation for now
+  const chain = RetrievalQAChain.fromLLM(llm, esStore.asRetriever(10));
```

Comment:
Thanks for bumping `k` from LangChain's default value of `4`, defined [here](https://github.com/langchain-ai/langchainjs/blob/044df7dad8c7120281d04f7a48b38b3257871b4f/langchain/src/vectorstores/base.ts#L73) to `10`